src/ontology/mondo-ingest.Makefile

```diff
@@ -575,6 +575,20 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+# update-repo-wrapper
+UPDATE_REPO_ODK_TAG := latest
```

Comment:
## `UPDATE_REPO_ODK_TAG`
Set to `latest` unless the user passes `ODK_TAG`, in which value is set to that.

`ODK_TAG` override usage:
- Works: `sh run.sh make ODK_TAG=v1.4.3 update-repo-wrapper`
- Fails: `ODK_TAG=v1.4.3 sh run.sh make update-repo-wrapper`. This doesn't cause the goal to fail, but the environmental variable doesn't get read.